tests/foreman/ui/test_errata.py

```diff
@@ -163,37 +169,88 @@ def vm(module_repos_collection_with_setup, rhel7_contenthost, target_sat):
     return rhel7_contenthost
 
 
+@pytest.fixture
+def registered_contenthost(
+    rhel_contenthost,
+    module_org,
+    module_lce,
+    module_target_sat,
+    repos=[CUSTOM_REPO_URL],
+):
+    """RHEL ContentHost registered in satellite,
+    Using SCA and global registration.
+
+    :param repos: list of upstream URLs for custom repositories,
+        default to CUSTOM_REPO_URL
+    """
+    content_view = module_target_sat.api.ContentView(
+        organization=module_org,
+        environment=[module_lce],
+    ).create()
```

Comment:
Can we go with `module_cv` here instead (the same one used in test body)?

Reply:
I had a problem promoting a published version of `module_cv`,  
I've implemented `module_published_cv` and promote the newest version after associating repos/content.